src/tiledb/cloud/geospatial/ingestion.py

```diff
@@ -962,6 +962,13 @@ def register_dataset_udf(
                 access_credentials_name=acn,
             )
 
+            # FIXME: This removes `dataset_type` metadata field for internal UI reasons
```

Comment:
We need to check if the output is an array - according to the discussions in the ticket, it will be a group for multi-LOD data.